plugins/hls-tactics-plugin/src/Wingman/LanguageServer.hs

```diff
@@ -136,8 +136,11 @@ properties :: Properties
    , 'PropertyKey "max_use_ctor_actions" 'TInteger
    , 'PropertyKey "timeout_duration" 'TInteger
    , 'PropertyKey "auto_gas" 'TInteger
+   , 'PropertyKey "proofstate_styling" 'TBoolean
    ]
 properties = emptyProperties
+  & defineBooleanProperty #proofstate_styling
```

Comment:
@berberman it's a little annoying that these properties need to be added in the opposite order as in the type. Is there a better pattern here?

Reply:
Hmm I think the order only affects in the type signature of `properties`, no?

Reply:
Yes, but in the opposite order in which they're defined. Notice that I'm adding to the top of `emptyProperties` here, but the resulting type is added to the _end_ of the type :(